src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <section class="search__box">
+      <form
+        class="search-bar search-bar__big"
+        data-qa="big"
+      >
+        <input
+          class="search-bar__search search-bar__search--large"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
Your class naming is a bit confusing. You have a `search-bar` class and a `search-bar__big` class on the same element. According to BEM methodology, `search-bar__big` would be an element inside `search-bar`, but in your case, they are on the same element. Consider renaming `search-bar__big` to `search-bar--big` to indicate that it is a modifier of `search-bar`.